src/pages/page-stop-signup.html

```diff
@@ -164,8 +164,14 @@
             * takes the user to the initiating hashtag page
             */
             _agree() {
-                this.path = sessionStorage.getItem('origin');
-                sessionStorage.clear();
+                if (typeof sessionStorage.getItem('origin') === 'string'
```

Comment:
Again is there any chance this would be something other than a string?

Reply:
Same as above, let's ask Xardas